### PR TITLE
Update supported currencies

### PR DESCRIPTION
**Pull request by wrboyce**
As per [the docs](https://raipay.io/docs), more than just XRB and USD is now supported by RaiPay.

### CODE REVIEW

**Comment by archseer** [comment]
I did the same :) https://github.com/coenjacobs/raipay-woocommerce/pull/1

File: src/WooCommerce/Plugin.php [hunk 1]
```diff
@@ -37,6 +72,6 @@ public function addGateway($array)
 	}
 
 	public function is_valid_for_use() {
-		return in_array( get_woocommerce_currency(), apply_filters( 'raipay_woocommerce_supported_currencies', array( 'USD', 'XRB' ) ) );
+		return in_array( get_woocommerce_currency(), apply_filters( 'raipay_woocommerce_supported_currencies', $supported_currencies ) );
```
Comment (archseer) [inline src/WooCommerce/Plugin.php:76]: XRB is now missing

Comment (wrboyce) [reply src/WooCommerce/Plugin.php:76]: Lol, nice catch - I just copy/pasted the table from the raipay docs and munged it in vim, didn't think that XRB wouldn't be in there. Fixed!

**Comment by wrboyce** [comment]
@archSeer whoops, didn't notice your PR sorry! I'm going to leave this PR open though as I think having the separate array is going to be easier to manage going forward if/when raipay add more currencies.

**Pull request closed without merging**